### ISSUE TITLE
Find in Selection checkbox does not have label

### ISSUE THREAD

**Issue by sonimishra** [microsoft/monaco-editor#1507]
User Experience:
If user navigate on connect Tab and label not define for radio and custom edit field, so it would difficult for user to know about the radio and edit field.

Repro Steps:
1- Navigate to url:- https://www.documentdb.com/sql/demo 
2- Home page will open.​
3- Navigate through all the controls appearing in "Home Page"
4- Navigate to Geospatial and select it.
5- Run Accessibility Insight for Web and verify the issue. 

Actual result:
Form elements doesn't contain labels

Expected result:
Form elements must have labels.

Issue: Form elements must have labels (label: https://dequeuniversity.com/rules/axe/3.1/label?application=axeAPI)

​
Target application title: Azure Cosmos DB Query Playground​
Target application url: https://www.documentdb.com/sql/demo​
​
Element path: #checkbox-0​
​
Snippet: input type="checkbox" class="checkbox" id="checkbox-0" disabled=""

How to fix:​
Fix any of the following:​
  aria-label attribute does not exist or is empty​
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty​
  Form element does not have an implicit (wrapped) <label>​
  Form element does not have an explicit <label>​
  Element has no title attribute or the title attribute is empty​
​
​
This accessibility issue was found using Accessibility Insights for Web 2.0.5 (axe-core 3.1.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
![MAS3 1 1_Form elements must have labels (#checkbox-0)](https://user-images.githubusercontent.com/30456910/61048898-cc890d80-a400-11e9-90fa-127df0cbfc1d.jpg)

**Comment by chrisdias** [comment]
this does not look like a Monaco editor bug, but instead an issue with the documentdb site.

**Comment by isidorn** [comment]
@sonimishra hi if you think this is an issue in the monaco editor please file the issue in concise steps in humand readable form only mentioning the monaco editor. Do not mention the Azure Stream Analyitics in your reproducable steps. Please put some effor in this.
Thank you for your understanding.

**Comment by ato9000** [comment]
![image](https://user-images.githubusercontent.com/8519407/66508895-d6b1bb00-ea86-11e9-88f0-83c338c585bd.png)

**Comment by chrisdias** [comment]
@isidorn it looks like a hidden checkbox on the Find widget, i'm not sure if this is something the user can actually get to?

![image](https://user-images.githubusercontent.com/1487073/66509877-b97dec00-ea88-11e9-8adf-cb16ad6c3add.png)

to repro (assuming chrome):
- go to https://chrome.google.com/webstore/detail/accessibility-insights-fo/pbjjkligggfmakdaogkfomddhfmpjeni and install the accessibility insights tool
- browse to https://www.documentdb.com/sql/demo
- click on the heart with the magnifier glass in the chrome bar, top right
- run "Fast Pass"
- note result that points to the `<input type="checkbox"...` and see the HTML in the picture or dev tools.

**Comment by alexdima** [comment]
This has been fixed in `master` where the Find in Selection is now a button like the other ones in the find widget.
